libxcodetools/src/main/kotlin/org/openbakery/assemble/AppPackage.kt

```diff
@@ -127,6 +130,32 @@ class AppPackage(
 		return File(applicationBundle.baseDirectory, "SwiftSupport")
 	}
 
+	fun addSymbols() {
+		val dSymPath = File(archive.absolutePath, "dSYMs")
+		if (dSymPath.exists()) {
+			logger.debug("Adding Symbols..")
+			tools.commandRunner.run("mkdir", "-p", "${applicationBundle.baseDirectory.absolutePath}/Symbols")
+			println(dSymPath.listFiles({ _, name -> name.endsWith(".dSYM")}).orEmpty().map { it.absolutePath })
+			for (dsym in dSymPath.listFiles({ _, name -> name.endsWith(".dSYM")}).orEmpty()) {
+				tools.commandRunner.run(
+					"xcrun",
+					"symbols",
+					"-noTextInSOD",
+					"-noDaemon",
+					"--arch",
+					"all",
+					"--symbolsPackageDir",
+					"${applicationBundle.baseDirectory}/Symbols",
+					dsym.absolutePath
+				)
+			}
+
+		} else {
+			logger.debug("Tried adding Symbols, but archive does not contain dSYMs to generate them.")
```

Comment:
If possible i think we could also bail out here.

Reply:
good idea, done! :)